### ISSUE TITLE
Tiled Galleries: editing a gallery does not show existing gallery images

### ISSUE THREAD

**Issue by jeherve** [Automattic/wp-calypso#30065]
### Steps to reproduce

I started from a Jetpack site:

1. I added a gallery to a new post. I uploaded 5 new images and a gallery was created.
2. Immediately after the upload was over, I clicked on the Pencil icon at the top of the block.
3. The Edit gallery modal came up, but I did not see the images I had just uploaded. Instead, I could only see all the other images I had already added to my site. Those images all appeared as selected like they were part of the gallery already.
4. Clicking on Update Gallery to close the modal effectively added all those images to that brand new gallery.

![image](https://user-images.githubusercontent.com/426388/50928313-a20ec580-145a-11e9-86d0-b637273b0ed0.png)

**Comment by sirreal** [comment]
We need to see whether this can be reproduced in core galleries and, if not, what they're doing differently.

**Comment by jeherve** [comment]
> We need to see whether this can be reproduced in core galleries

You're right, that's a problem with core galleries as well. Closing as duplicate of https://github.com/WordPress/gutenberg/issues/9072

**Comment by sirreal** [comment]
We might keep this open. If this is a Gutenberg bug, we're shipping the same issue and we'll need the same fix. Tiled Gallery duplicates a lot of the same code as core.
